extensions/transform/org.eclipse.smarthome.transform.scale/src/main/java/org/eclipse/smarthome/transform/scale/internal/ScaleTransformationService.java

```diff
@@ -47,9 +47,6 @@
      *            the list of properties defining all the available ranges
      * @param source
      *            the input to transform
-     *
```

Comment:
Here I will remove the <p> in the beginning, as well as edit the JavaDoc according to your comment. In general, in the upcoming days, I will create another PR for fixing all of the unclosed <p> tags in JavaDocs throughout ESH - this has to be done. :)